### PR TITLE
Reset all form validations upon MH registration submission

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity###

*Description of changes:*
- Added function resetAllValidations to reset the forms along the stepper once the MH registration is submitted

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good to me.

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue [hunk 1]
```diff
@@ -184,6 +184,27 @@ export default defineComponent({
...
+      setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.REF_NUM_VALID, false)
+
+      // Reset home owner validations
+      setValidation(MhrSectVal.HOME_OWNERS_VALID, MhrCompVal.OWNERS_VALID, false)
+
+      // Reset home location validations
+      setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LOCATION_TYPE_VALID, false)
+      setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.CIVIC_ADDRESS_VALID, false)
+    }
+
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: Looks good but could you move this into the validation composable?

Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: useMhrValidations

**Review by cameron-eyds — COMMENTED** [review]
Good job! Just 1 comment

File: ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: For sure! Would that composable file be "useMhrValidations.ts"?

Comment (owillborn) [reply ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue]: Gotcha thanks!

**Review by ChasBoucher — APPROVED** [review]
Other than Camerons comments.  Looks Good.

**Review by dimak1 — CHANGES_REQUESTED** [review]
As per Cameron's comment, please move validation to a composable. Thanks!

**Review by cameron-eyds — APPROVED** [review]
Looks good, thank you!

**Review by dimak1 — APPROVED** [review]

**Pull request merged**